frontend/src/components/maps/leaflet/mapUtils.tsx

```diff
@@ -244,13 +244,16 @@ export const pointToLayer = (feature: ICluster, latlng: LatLngExpression): Layer
  * Get an icon type for the specified cluster property details (type, draft, erp, spp etc)
  */
 export const getMarkerIcon = (feature: ICluster, selected?: boolean) => {
-  const { propertyTypeId, projectWorkflow, classificationId } = feature?.properties;
+  const { propertyTypeId, projectWorkflow, classificationId, parcelDetail } = feature?.properties;
   if (propertyTypeId === PropertyTypes.DRAFT_PARCEL) {
     return draftParcelIcon;
   } else if (propertyTypeId === PropertyTypes.DRAFT_BUILDING) {
     return draftBuildingIcon;
   } else if (selected) {
-    if ([Workflows.ERP, Workflows.ASSESS_EX_DISPOSAL].includes(projectWorkflow)) {
+    if (
```

Comment:
There's something funky going on with selecting markers, the shape of the data is inconsistent, spent some time debugging but need to work on my other tasks. This does "solve" the issue though. `parcelDetail` is normally stored on marker click, but for some reason selecting certain markers they instantly deselect (even though select is set to true) so it would then have to look at the `projectWorkflow` instead of the `parcelDetail`. Creating a separate bug as these were found during my story to add project number to the panel..

Reply:
mikaela noticed that when clicking on a pin it will change to blue/green no matter what colour it used to be. Would this fix that?

Reply:
Yes this will! @agalbrai